exchanges/exchange.go

```diff
@@ -799,3 +801,18 @@ func (e *Base) DisableRateLimiter() error {
 func (e *Base) EnableRateLimiter() error {
 	return e.Requester.EnableRateLimiter()
 }
+
+// KlineIntervalEnabled returns if requested interval is enabled on exchange
+func (e *Base) KlineIntervalEnabled(in kline.Interval) bool {
+	return e.Features.Enabled.KlineCapabilities.Intervals[in.Word()]
+}
+
+func (e *Base) KlineValidFeatures() bool {
+	return false
+}
+
+// FormatExchangeKlineInterval returns Interval to string
+// Exchanges can override this if they require custom formatting
+func (e *Base) FormatExchangeKlineInterval(in kline.Interval) string {
+	return common.ErrNotYetImplemented.Error()
```

Comment:
implementing this?

Reply:
Originally I didn't want to implement this inside exchange as each exchange can override it but I have decided against that and now it will now just return String() with each exchange still being able to override it

Reply:
Okay! 👍